core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -2566,7 +2566,7 @@ def _resetMultipleFileHandlerClassVars():
         c.prior_validated_feature_ids = None
         c.prior_validated_header = None
 
-# --------------------------- resource definition wise test ------------------------------
+# --------------------------- resources wise test ------------------------------
```

Comment:
i looked a bit in this file - im not entirely sure why these tests are often called `wise` test and not e.g. ResourceTest. Again this is following existing convention so prolly fine, but that wording strikes me as a bit odd